src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/WPF/AppDotXamlDocument.cs

```diff
@@ -0,0 +1,944 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Xml;
+using Microsoft.VisualStudio.TextManager.Interop;
```

Comment:
There are 11 qualified references to `System.Diagnostics.Debug` in this file. Perhaps:

```suggestion
using Microsoft.VisualStudio.TextManager.Interop;
using Debug = System.Diagnostics.Debug;
```